functional_testing/Openvcloud/utils/utils.py

```diff
@@ -456,6 +457,21 @@ def get_vm_connection(self, vm_id, wait_vm_ip=True, password=None, login=None, p
                 continue
         return connection
 
+    def get_vm_ssh_client(self, vm_id, vm_ip=None , password=None, login= None):
```

Comment:
- get_vm_ssh_client --> get_vm_public_ssh_client
- why vm_ip is not required param?